### ISSUE TITLE
On the Build method, accept parameters for the user to choose his custom preferences

### ISSUE THREAD

**Issue by DanielCauser** [BSiLabs/HttpTracer#3]
Preferences such as:
- Select color of the output messages
- Turn the logging on and off

**Comment by ChaseFlorell** [comment]
I've done some preliminary digging, and there's no way to automagically change the color of the text in the output window.

Initially I tried this, but it didn't affect the output

```
Console.ForegroundColor = ConsoleColor.Green;
Console.WriteLine(myText);
Console.ResetColor();
```

If you still want this functionality, it'll definitely work in a console app (exe), but won't make any difference to the Output window.

Also, I took a run at #1 and found that we can't do that either since the libs required are not compatible with iOS or Android. I'm not even sure how we'd write a facade to achieve it.
